packages/turf-bbox/README.md

```diff
@@ -4,7 +4,7 @@
 
 ## bbox
 
-Takes a set of features, calculates the bbox of all input features, and returns a bounding box.
+Calculates the bounding box for any GeoJSON object, including FeatureCollection.
```

Comment:
This file is auto-generated so it doesn't actually need fixing, but it doesn't matter if you've done it :)